UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -167,15 +167,15 @@ public void LengthTimesSpecificWeightEqualsPressure()
         public void ToStringReturnsCorrectNumberAndUnitWithDefaultUnitWhichIsMeter()
         {
             var meter = Length.FromMeters(5);
-            string meterString = meter.ToString(CultureInfo.InvariantCulture);
+            string meterString = meter.ToString(null, CultureInfo.InvariantCulture);
```

Comment:
This I don't like, c# users expect certain ToString() signatures to exist and having to use named parameters or pass `null` is awkward.